### ISSUE TITLE
Did you repeat the square operation?

### ISSUE THREAD

**Issue by XueruiSu** [zoubohao/DenoisingDiffusionProbabilityModel-ddpm-#12]
![image](https://user-images.githubusercontent.com/77718956/211013276-663b1aba-f443-4485-a9cd-b5d4433ee19d.png)
F.mseloss can do the square operation, so may we not need to do the "**2." operation?
![image](https://user-images.githubusercontent.com/77718956/211013508-cca3ceaf-22c9-4b8e-9f5e-6243ce53e5bc.png)
the position of the code is in the file named TrainCondition.py on the directory of DiffusionFreeGuidence

**Comment by zoubohao** [comment]
hhhhh, the ** 2 is for "b", batch size, not for the loss. Since the reduction parameter in F.mse_loss sets None, then the loss would be huge if we only use .sum()..... It is difficult for training and I prefer to use .sum() than .mean(). b ** 2 is used to control the scale of loss.

**Comment by XueruiSu** [comment]
thanks for reply
